### ISSUE TITLE
Warning that files are not getting purged in production

### ISSUE THREAD

**Issue by hitchho** [cjkpl/oc-tabularasa-theme#1]
Getting following message when running npm run prod

   ������  Tailwind is not purging unused styles because no template paths have been provided.
      If you have manually configured PurgeCSS outside of Tailwind or are deliberately not
      removing unused styles, set `purge: false` in your Tailwind config file to silence
      this warning.

what do I need to do to set the paths?

**Comment by cjkpl** [comment]
Hi @Hitchhooker, 

Thanks for your note. I think it actually *is* purging, despite the messages- just compare the size in dev and prod builds.
I'll see what causes it over the next couple of days.

**Comment by hitchho** [comment]
using purge through tailwind.config.js got rid of the warning but as you said, it did purge regardless of those warnings.
```
purge: [
    './pathto/**/*.htm',
    './pathto/**/*.css',
    './pathto/**/*.js',
  ],
```

**Comment by cjkpl** [comment]
Resolved with release 1.0.1 - removed purgeCSS config from laravel mix manifest, switched to TailwindCSS' internal support for purgeCSS
